src/views/v2/list/nav/prev-disabled.php

```diff
@@ -9,19 +9,20 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.10
+ * @version TBD
  *
  */
 ?>
 <li class="tribe-events-c-nav__list-item tribe-events-c-nav__list-item--prev">
-	<button class="tribe-events-c-nav__prev tribe-common-b2 tribe-common-b1--min-medium" disabled>
-		<?php
-			$events_label = '<span class="tribe-events-c-nav__prev-label-plural"> ' . tribe_get_event_label_plural() . '</span>';
-			echo wp_kses(
-				/* translators: %s: Event (plural or singular). */
-				sprintf( __( 'Previous %1$s', 'the-events-calendar' ), $events_label ),
-				[ 'span' => [ 'class' => [] ] ]
-			);
-		?>
-	</button>
+	<button 
+		class="tribe-events-c-nav__prev tribe-common-b2 tribe-common-b1--min-medium" 
+		disabled
+	><?php
+		$events_label = '<span class="tribe-events-c-nav__prev-label-plural">' . tribe_get_event_label_plural() . '</span>';
+		echo wp_kses(
+			/* translators: %s: Event (plural or singular). */
+			sprintf( __( 'Previous %1$s', 'the-events-calendar' ), $events_label ),
+			[ 'span' => [ 'class' => [] ] ]
+		);
+	?></button>
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 2 tabs, found 1
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 36688eb2f21be19db0f3527d42ea8dac</sub>